### PR TITLE
Added Spreaker Studio to Closed Source apps list

### PR DESCRIPTION
**Pull request by pracucci**
I'm a Spreaker engineer. We recently launched "Spreaker Studio for Desktop", an audio recording and broadcasting app for OSX and Windows. Would be great to list it too.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ Some good apps written with Electron.
 - [Stremio](http://www.strem.io) - Next generation media center.
 - [RESTer](http://getrester.com) - Easy and beautiful way to test your APIs and servers.
 - [Typetalk](http://www.typetalk.in) - Share and discuss ideas with your team through instant messaging.
-
+- [Spreaker Studio](https://www.spreaker.com/desktop-studio/download) - Audio recording and broadcasting app from Spreaker.
```
Comment (shinnn) [inline readme.md]: `Audio recording and broadcasting app from Spreaker.`

↓

`Audio recording and broadcasting.`

Comment (shinnn) [inline readme.md]: `https://www.spreaker.com/desktop-studio/download` is a direct link to the `dmg` file. Replace it with the introduction page of the app.

**Comment by shinnn** [comment]
You're missing some of the contribution guidelines. https://github.com/sindresorhus/awesome-electron/blob/d09b6b9c663355b9f4c68cfc0ba741387f04bfcd/contributing.md

File: readme.md [hunk 2]
```diff
@@ -68,7 +68,7 @@ Some good apps written with Electron.
 - [Stremio](http://www.strem.io) - Next generation media center.
 - [RESTer](http://getrester.com) - Easy and beautiful way to test your APIs and servers.
 - [Typetalk](http://www.typetalk.in) - Share and discuss ideas with your team through instant messaging.
-
```
Comment (arthurvr) [inline readme.md]: Don't remove this line.

**Comment by sindresorhus** [comment]
ping @pracucci 

-

*Don't open another PR. Just update this one.*

**Comment by pracucci** [comment]
Thank you very much for your feedback. We currently don't have a dedicated page for the app, however we plan to build it in a couple of weeks. For this reason I'm waiting for it and then I will update the PR with all fixes.

Does it make sense to you?

**Comment by sindresorhus** [comment]
Sure

**Comment by sindresorhus** [comment]
Closing for now. Just ping me here when ready and I'll reopen.

**Comment by pracucci** [comment]
I'm ready to update the PR. May you reopen it, please?

**Comment by arthurvr** [comment]
Sure.

**Comment by sindresorhus** [comment]
This was already merged in https://github.com/sindresorhus/awesome-electron/commit/eba56d94a44a1f461afb05fd44146cc069441d70.

**Pull request closed without merging**